src/pages/MyTransfers/TransfersTable.js

```diff
@@ -0,0 +1,229 @@
+                ? (column.renderer
+                  ? column.renderer(row[column.name])
+                  : row[column.name])
+                : '--';
+
+              return (
+                <OverflownCell
+                  key={cellKey}
+                  cellValue={cellValue}
+                  cellColor={cellColor}
+                >
+                  {cellValue}
+                </OverflownCell>
+              );
+            })}
+          </TableRow>
+        );
+      })}
+    </TableBody>
+  );
+
+};
+
+
+/**@function
+ * @name TransfersTable
+ * @description Renders the transfers table
+ * @param {string} tableTitle Name of the table to be displayed
+ * @param {object} tableColumns Array of table column objects to be displayed
+ * @param {object} tableRows Array of table row objects to be displayed
+ *
+ * @returns {JSX.Element} - transfer table component
+ */
+const TransfersTable = ({
+                          tableTitle,
+                          tableRows,
+                          totalRowCount,
+                        }) => {
+
+  // get data from context
+  const { pagination, setPagination, statusList, tableColumns } = useTransfersContext();
+
+  // pagination
+  const [page, setPage] = useState(0);
+  const [rowsPerPage, setRowsPerPage] = useState(10);
+  const handleRowsPerPageChange = (e) => {
+    const newRowsPerPage = parseInt(e.target.value, 10);
+    setRowsPerPage(newRowsPerPage);
+    setPage(0);
+
+    const newPagination = {
+      limit: newRowsPerPage,
+      offset: 0,
+    };
+    setPagination(newPagination);
+  };
+
+  const handlePageChange = (e, newPage) => {
+    setPage(newPage);
+    const newPagination = { ...pagination, offset: newPage * rowsPerPage };
+    setPagination(newPagination);
+  };
+
+  // get color corresponding to the status value, else default color
+  const getStatusColor = (status) => {
+    const color = statusList.find(x => x.value === status).color;
+    return color ? color : '#585B5D';
+  };
+
+  return (
+    <Grid container direction={'column'}>
+      <TransfersTableHeader
+        tableTitle={tableTitle}
+        getStatusColor={getStatusColor}
+      />
+
+      <TableContainer component={Paper} sx={{ height: '400px', width: '1000px' }}>
+        <Table stickyHeader sx={{ minWidth: 650 }} aria-label='transfers table' data-testid='transfers-table'>
+          <TableHead>
+            <TableRow>
+              {tableColumns.map((column, id) => {
+                return (
+                  <TableCell key={`${id}-${column.description}`}
+                             sx={{ fontSize: '14px' }}>{column.description}</TableCell>
+                );
+              })}
+            </TableRow>
+          </TableHead>
+          <TransfersTableBody
+            tableColumns={tableColumns}
+            tableRows={tableRows}
+            getStatusColor={getStatusColor}
+          />
+        </Table>
+      </TableContainer>
+
+      <TablePagination
```

Comment:
There is an issue in pagination related to a count: 
![pagination](https://github.com/Greenstand/treetracker-wallet-admin-client/assets/19413810/2fefebd7-c609-44a8-ba01-14823ea7eb29)

Reply:
Also, there a several console errors which might be related to this: 
![console errors](https://github.com/Greenstand/treetracker-wallet-admin-client/assets/19413810/3d086240-3cbd-45da-bff7-e9c924251655)

Reply:
Yes this depends on this issue: https://github.com/Greenstand/treetracker-wallet-api/issues/385.

Reply:
In this case, I would better comment rendering of the pager with a comment like 'TODO: uncomment when API fix #385 is ready' and show it again as soon as we can do it correctly